src/project-tests/technical-docs-tests.js

```diff
@@ -287,7 +287,7 @@ export default function createTechnicalDocsPageTests() {
           el.firstElementChild.innerText.trim()
         );
         const linkText = classArray('nav-link').map((el) =>
-          (/[^\n\t\f\r\v]+/).exec(el.innerText)[0]
+          (/[^\n\t\f\r\v]+/).exec(el.textContent)[0]
```

Comment:
`textContent` will return the value of hidden elements so I'm not sure this is what we want. If the nav links are hidden, the project shouldn't really pass the tests.

If the goal is to ignore case, I think a better approach would be to just "normalize" both values by either making them all upper case or lower case, and then doing the comparison on the normalized values.